scanner/metadata/metadata_internal_test.go

```diff
@@ -64,14 +64,22 @@ var _ = Describe("Tags", func() {
 	})
 
 	Describe("getAllTagValues", func() {
-		It("returns values from all tag names", func() {
+		It("returns genre values from all tag names", func() {
 			md := &Tags{}
 			md.tags = map[string][]string{
 				"genre": {"Rock", "Pop", "New Wave"},
 			}
 
 			Expect(md.Genres()).To(ConsistOf("Rock", "Pop", "New Wave"))
 		})
+		It("returns publisher values from all tag names", func() {
```

Comment:
This test is a duplicate of the previous one, and is not testing the dedup logic you implemented. Also, I think we should rename these two `It`s. Maybe something like:

```go
    It("returns values from all tag names")
    
    and
    
    It("remove duplicated values")
```

Reply:
Done